benchmarks/user/advisor/roofline.py

```diff
@@ -101,6 +110,13 @@ def roofline(name, project, scale, precision, mode, th):
     gflops_min = 2**0
     width = ai_max
 
+    # Declare the dictionary that will hold the JSON information
+    roofline_data = {}
+
+    # Declare the two types of rooflines dictionaries
```

Comment:
so this is to generate the four lines L1/L2/L3/DRAM? and likewise for compute (I don't remember what they are, but you get the point)?

Reply:
Yes, for compute...sp_flops, dp_flops...etc